mlflow/sklearn/__init__.py

```diff
@@ -591,15 +667,43 @@ def _dump_model(pickle_lib, sk_model, out):
             raise
 
 
-def _save_model(sk_model, output_path, serialization_format):
+def _save_model(sk_model, output_path, serialization_format, skops_trusted_types):
```

Comment:
NOTE: Require user to set `skops_trusted_types` when saving model, and checks whether the model can be loaded back by the provided `skops_trusted_types`, if the model can't be loaded back by these provided `skops_trusted_types`, raise error in `_save_model`.